src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+    left: 50px;
+
+    background-image: url(../images/logo.png);
```

Comment:
Using background images for logos is not the best approach. There are 2 reasons for that:
- Using <img /> tag will be useful when you need to use CSS media queries and display a different logo size depending on the screen resolution.
- <img /> tag is better for accessibility reasons. First of all, users will be able to view the alternative text when images are disabled.
Thus, background images should be used only if the picture is just a background, not a part of the information shown on the page